.github/actions/upload-nightly/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Upload Nightly"
+description: "Upload file to the nightly repository"
+
+inputs:
+  repository_url:
+    description: "The URL of the repository"
+    required: true
+  repository_username:
```

Comment:
We are currently using an environment for this, we can switch to an input but better do it for all workflows later.